digital-deck-server/index.js

```diff
@@ -5,8 +5,12 @@ const http = require("http");
 const { Server } = require('socket.io');
 const sessionRoute = require('./routes/session');
 const playerRoute = require('./routes/player');
+const {handleSocket} = require('./socket');
 
 const app = express();
+
+app.use(express.static('../digital-deck-ui')); 
```

Comment:
What does this do?

Reply:
it makes it possible to access files from this folder via HTTP. It was done in all socket tutorials, I don't know if we require it or not